ci-operator/step-registry/openstack-k8s-operators/test/build-deploy/openstack-k8s-operators-test-build-deploy-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: openstack-k8s-operators-test-build-deploy
+  steps:
```

Comment:
To make this simple, you could maintain only:
    test:
    - ref: openstack-k8s-operators-build
    
since your test is on "...build-commands.sh" script.

Reply:
right , tnx 
changing this